src/sentry/uptime/config_producer.py

```diff
@@ -25,16 +29,30 @@ def _get_producer() -> KafkaProducer:
 _configs_producer = SingletonProducer(_get_producer)
 
 
-def produce_config(config: CheckConfig):
-    _produce_to_kafka(UUID(config["subscription_id"]), UPTIME_CONFIGS_CODEC.encode(config))
+def produce_config(destination_region_slug: str, config: CheckConfig):
+    _produce_to_kafka(
+        destination_region_slug,
+        UUID(config["subscription_id"]),
+        UPTIME_CONFIGS_CODEC.encode(config),
+    )
+
 
+def produce_config_removal(destination_region_slug: str, subscription_id: str):
+    _produce_to_kafka(destination_region_slug, UUID(subscription_id), None)
 
-def produce_config_removal(subscription_id: str):
-    _produce_to_kafka(UUID(subscription_id), None)
 
+def _produce_to_kafka(
+    destination_region_slug: str, subscription_id: UUID, value: bytes | None
+) -> None:
+    region_config = get_region_config(destination_region_slug)
+    if region_config is None:
+        logger.error(
+            "Attempted to create uptime subscription with invalid region slug",
```

Comment:
nit: I've typically tried to make these more "machine like", like `uptime_config_producer.produced_with_invalid_region_slug` or whatever

Reply:
I feel like that's going to make the error message in sentry itself less readable though... I can see it being nicer for actual logging